lib/authentication/view/sign_up_view.dart

```diff
@@ -45,25 +51,43 @@ class _SignUpViewState extends State<SignUpView> {
     _fetchFilter();
   }
 
+  void parse(TextEditingController email, TextEditingController firstName,
+      TextEditingController lastName) {
+
+    String emailName = email.text.replaceAll(RegExp(r'[0-9]'), '');
+    List<String> name = emailName.split('.');
+    List<String> firstNameString;
+
+    if (name[0].indexOf('_') < 0) {
+      firstName.text = name[0].titleCase;
+    } else {
+      firstNameString = name[0].split('_');
+      firstName.text =
+          firstNameString[0].titleCase + " " + firstNameString[1].titleCase;
```

Comment:
Prefer using apostrophes over quotation marks:
```suggestion
          firstNameString[0].titleCase + ' ' + firstNameString[1].titleCase;
```